chef/instrumentald/templates/default/instrumentald.toml.erb

```diff
@@ -11,3 +11,13 @@ system = ["cpu", "disk", "load", "memory", "network", "swap"]
 # mysql = ["root@tcp(127.0.0.1:3306)/"]
 # nginx = ["http://localhost:80/status"]
 # postgresql = ["postgres://postgres@localhost?sslmode=disable"]
+
+
+debug = true
+
+# Setup some hosts to test with serverspec
+mongodb = [
+  "mongodb://localhost:27017",
+  "mongodb://instrumentald-atlas-test:<PASSWORD>@cluster0-shard-00-00-uvipm.mongodb.net:27017,cluster0-shard-00-01-uvipm.mongodb.net:27017,cluster0-shard-00-02-uvipm.mongodb.net:27017/<DATABASE>?ssl=true&replicaSet=Cluster0-shard-0&authSource=admin",
```

Comment:
I'm thinking about whether we should cut these down to a real-like but not-real uri, but I'm not sure it actually matters, other than we may be putting some extra connections to a non existent cluster out there.

username:password@cluster-shard-00-00-test.mongodb.net or something.

Probably, it doesn't matter.